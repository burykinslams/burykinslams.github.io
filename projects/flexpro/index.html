<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=970,user-scalable=yes">
	<meta name="format-detection" content="telephone=no">
	<title>flex-pro</title>
	<link href="css/screen.css" rel="stylesheet" media="all" type="text/css">
	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,700,600&subset=latin,cyrillic-ext' rel='stylesheet' type='text/css'>
</head>
<body>
<!--begin header-->
	<div class="header">
		<div class="top-line clearfix">
			<div class="container">
				<ul class="left">
					<li><a href="#">Доставка</a></li>
					<li><a href="#">Оплата</a></li>
					<li><a href="#">Дисконтная программа</a></li>
					<li><a href="#">Контакты</a></li>
					<li><a href="#">Топ-50</a></li>
					<li><a href="#">Акции</a></li>
				</ul>
				<ul class="user right">
					<li><a href="#" class="pop-up-open"><i class="reg"></i>Регистрация</a></li>
					<li><a href="#"><i class="login"></i>Вход</a></li>
				</ul>
			</div>
		</div>
		<div class="head">
			<div class="container">
				<div class="head-item logo">
					<a href="#"><img src="img/logo.jpg" alt="logo"></a>
				</div>
				<div class="head-item contact">
					<h2><i></i>( 050 ) 732-60-98</h2>
					<p>с 10.00 до 22.00 ежедневно</p>
					<a href="#">Заказать обратный звонок</a>
				</div>
				<div class="head-item search">
					<form action="">
						<input type="search" placeholder="Поиск...">
						<input type="submit" value="">
					</form>
				</div>
				<div class="head-item basket">
					<a href="#"><i></i>Корзина</a>
					<p>Товров: <span>0 шт.</span></p>
					<p>На сумму: <span>0,0 грн.</span></p>
				</div>
			</div>
		</div>
		<div class="nav clearfix">
			<div class="container">
				<ul>
					<li><a href="#">Спортивное питание</a></li>
					<li><a href="#">Для снижения веса</a></li>
					<li><a href="#">Витамины и минералы</a></li>
					<li><a href="#">Повышение тестостеронa</a></li>
					<li><a href="#">Энергетики</a></li>
					<li><a href="#">Экипировка</a></li>
					<li><a href="#">Аксесуары</a></li>
					<li><a href="#">Производители</a></li>
				</ul>
			</div>
		</div>
	</div>
<!--end header-->
<!--begin slider-->
	<div class="slider">
		<div class="container">
			<div class="slider-top js-slider-top">
				<div>
					<div class="slider-item">
						<p>Количество/размер: 300 g.<br>Страна-производитель: США<br>Форма: порошок<br>
						<span>99,99 <sub>грн</sub></span> 
						</p>
						<a href="" class="slider-btn">Купить сейчас</a>
					</div>
				</div>
				<div>
					<div class="slider-item">
						<p>Количество/размер: 300 g.<br>Страна-производитель: США<br>Форма: порошок<br>
						<span>99,99 <sub>грн</sub></span>
						</p>
						<a href="" class="slider-btn">Купить сейчас</a>
					</div>
				</div>
			</div>
			<!--begin slider-logo-->
			<div class="slider-logo js-slider-logo">
				<div class="slider-logo-item">
					<img src="img/logo1.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo2.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo3.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo4.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo5.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo1.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo2.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo3.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo4.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo2.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo3.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo4.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo3.jpg" alt="">
				</div>
				<div class="slider-logo-item">
					<img src="img/logo4.jpg" alt="">
				</div>
			</div>
			<!--end slider-logo-->
		</div>
	</div> 
<!--end slider-->
<!--begin content-->
<div class="content">
	<div class="container clearfix">
		<!--begin content-wrap-->
		<div class="content-wrap left">
			<div class="topic clearfix">
				<h2 class="week-product left">Товар недели</h2>
				<a href="#" class="all-product right">Все товары</a>
			</div>
			<div class="content-item">
				<img src="img/new.png" class="new">
				<img src="img/content-item.png" alt="">
				<a href="#">100% Casein Protein 400 г.</a>
				<div class="sold clearfix">
					<div class="item-info">
						<a href="#" class=".remind">Сообщить когда появится</a>
					</div>
					<a href="#" class="sold-btn">Продан<i></i></a>
				</div>
			</div>
			<div class="content-item">
				<img src="img/new.png" class="new">
				<img src="img/content-item.png" alt="">
				<a href="#">100% Casein Protein 400 г.</a>
				<div class="buy clearfix">
					<div class="item-info">
						<p class="old-price">1888 <sub>грн.</sub></p>
						<p class="price">1826 <sub>грн.</sub></p>
					</div>
					<a href="#" class="buy-btn">Купить<i></i></a>
				</div>
			</div>
			<div class="content-item">
				<img src="img/new.png" class="new">
				<img src="img/content-item.png" alt="">
				<a href="#">100% Casein Protein 400 г.</a>
				<div class="buy clearfix">
					<div class="item-info">
						<p class="old-price">1888 <sub>грн.</sub></p>
						<p class="price">1826 <sub>грн.</sub></p>
					</div>
					<a href="#" class="buy-btn">Купить<i></i></a>
				</div>
			</div>
			<div class="content-item">
				<img src="img/new.png" class="new">
				<img src="img/content-item.png" alt="">
				<a href="#">100% Casein Protein 400 г.</a>
				<div class="buy clearfix">
					<div class="item-info">
						<p class="old-price">1888 <sub>грн.</sub></p>
						<p class="price">1826 <sub>грн.</sub></p>
					</div>
					<a href="#" class="buy-btn">Купить<i></i></a>
				</div>
			</div>
			<div class="content-item">
				<img src="img/new.png" class="new">
				<img src="img/content-item.png" alt="">
				<a href="#">100% Casein Protein 400 г.</a>
				<div class="buy clearfix">
					<div class="item-info">
						<p class="old-price">1888 <sub>грн.</sub></p>
						<p class="price">1826 <sub>грн.</sub></p>
					</div>
					<a href="#" class="buy-btn">Купить<i></i></a>
				</div>
			</div>
			<div class="content-item">
				<img src="img/new.png" class="new">
				<img src="img/content-item.png" alt="">
				<a href="#">100% Casein Protein 400 г.</a>
				<div class="buy clearfix">
					<div class="item-info">
						<p class="old-price">1888 <sub>грн.</sub></p>
						<p class="price">1826 <sub>грн.</sub></p>
					</div>
					<a href="#" class="buy-btn">Купить<i></i></a>
				</div>
			</div>
			<!--begin content-info-wrap-->
			<div class="content-info-wrap left">
				<div class="top-popular">  
					<h2>Самое популярное</h2>
					<table>
					<tr>
						<td>
							<a href="#">1. Power System L-Carnitin 60 000</a> 
							<p>Витаминоподобное вещество, аналогичное тому, которое вырабатывается печенью организма человека для вовлечения...</p>     
						</td>
						<td>     
							<a href="#">Протеин</a>
							<p>Основной компонент спортивного питания. Обеспечивает организм белками, помогая нарастить мышечную массу.</p>
						</td>
						<td>
							<a href="#">Optimum Nutrition</a>     
							<img src="img/top-pop-img.jpg" alt="">
						</td>
					</tr>
					<tr>
						<td>
							<a href="#">2. Optimum Nutrition 100% Whey GS</a>
							<a href="#">3. Universal Nutrition Daily Formula</a>
							<a href="#">4. Optimum Nutrition 100% Whey GS</a>
							<a href="#">5. Optimum Nutrition Serious Mass</a>
						</td>
						<td>
							<a href="#">Жиросжигатели/Снижен</a>
							<a href="#">Аминокислоты</a>
							<a href="#">Здоровье и Долголетие</a>
							<a href="#">Заменители питания</a>    
						</td>
						<td>
							<a href="#">Power System</a>
							<a href="#">Weider</a>
							<a href="#">Universal Nutrition</a>
							<a href="#">Dymatize Nutrition</a>
						</td>
					</tr>
					</table>
				</div> 
				<div class="article">
					<h2>Спортивное питание</h2>
					<p>
					Говорить о том, что спортивное питание – главная составляющая успеха, сегодня излишне, так как об этом знает даже начинающий спортсмен. Сотни научных лабораторий во всем мире проводят большую работу по разработке и испытанию препаратов, которые впоследствии ежедневно  принимают бодибилдеры, пауэрлифтеры, фитнесмены  с одной целью - обрести идеальное тело и здоровый дух. Это  концентраты органических веществ и комплексные пищевые добавки, призванные выполнять следующие задачи: ускорение процесса увеличения мышечной массы, сжигание излишних жиров, поддержка здоровья спортсмена в целом.
					</p>
				</div>
			</div> 
			<!--end content-info-wrap-->
		</div>
		<!--end content-wrap-->
		<!--begin side-bar--> 
		<div class="side-bar right">
			<div class="s-bar-item history clearfix">
				<h3>Вы смотрели</h3>
				<div class="sub-item">
					<img src="img/h-item1.png" alt="">
					<a href="#">Heavy GASP Tee, Black GASP</a>
					<p>Мужская одежда</p>
				</div>
				<div class="sub-item">
					<img src="img/h-item2.png" alt="">
					<a href="#">L-Carnitin 60 000 Power System</a>
					<p>Спортивное питание</p>
				</div>
				<p  class="all"><a href="#">Вся история</a></p>
			</div>
			<div class="s-bar-item news clearfix">
				<h3>Новости</h3>
				<div class="sub-item">
					<time>15 апреля 2014</time>
					<a href="#">BIKINI-MENS STAR-2014</a>
				</div>
				<div class="sub-item">
					<time>15 апреля 2014</time>
					<a href="#">Проверь свою силу! #57 ЖЕЛЕЗНЫЙ РЕЙТИНГ</a>
				</div>
				<p  class="all"><a href="#" class="all">Все новости</a></p>
			</div>
			<div class="s-bar-item delivery">
				<h3>Рассылка</h3>
				<form action="">
					<label for="">Подпишить на новости и скидки!</label>
					<input type="email" placeholder="Ваш e-mail">
					<input type="submit" value="Ок">
				</form>
			</div>
			<div class="social">
				<img src="img/soc1.png">
				<img src="img/soc2.png" alt="">
			</div>
		</div>
		<!--end side-bar-->
	</div>
</div> 
<!--end content-->
<!--begin footer-->
<div class="footer">
	<div class="bot-line">
		<div class="container">
			<p class="copyright"><span>FLEX PRO</span> © Сеть магазинов спортивного питания</p>
		</div>
	</div>
	<div class="bot-nav">
		<div class="container">
			<div class="b-nav-item info-list">
				<ul>
					<li><a href="#">Покупателю</a>
					<li><a href="#">Скидки</a></li>
					<li><a href="#">Доставка</a></li>
					<li><a href="#">Способы оплаты</a></li>
					<li><a href="#">Адреса магазинов</a></li>
				</ul>
			</div>
			<div class="b-nav-item info-list">
				<ul>
					<li><a href="#">Ассортимент</a></li>
					<li><a href="#">Новинки</a></li>
					<li><a href="#">Бестселлеры</a></li>
					<li><a href="#">Каталог товаров</a></li>
				</ul>
			</div>
			<div class="b-nav-item info-list">
				<ul>
					<li><a href="#">Компания</a></li>
					<li><a href="#">Контакты</a></li>
					<li><a href="#">Информация</a></li>
					<li><a href="#">Вакансии</a></li>
					<li><a href="#">Адреса магазинов</a></li>
				</ul>
			</div>
			<div class="b-nav-item contact">
					<h2><i></i>( 050 ) 732-60-98</h2>
					<p>с 10.00 до 22.00 ежедневно</p>
					<a href="#">Заказать обратный звонок</a>
			</div>
			<div class="b-nav-item banners">
				<a href="#"><img src="img/bot-ban1.png"></a>
				<a href="#"><img src="img/bot-ban2.png"></a>
				<a href="#"><img src="img/bot-ban3.png"></a>
			</div>
		</div>
	</div>
	<div class="bot-social clearfix">
		<div class="container">
			<div class="pay-forms left">
				<a href="#"><img src="img/pay1.png" alt=""></a>
				<a href="#"><img src="img/pay2.png" alt=""></a>
				<a href="#"><img src="img/pay3.png" alt=""></a>
				<a href="#"><img src="img/pay4.png" alt=""></a>
				<a href="#"><img src="img/pay5.png" alt=""></a>
			</div>
			<div class="networks right">
				<a href=""><i class="soc1"></i></a>
				<a href=""><i class="soc2"></i></a>
				<a href=""><i class="soc3"></i></a>
				<a href=""><i class="soc4"></i></a>
				<a href=""><i class="soc5"></i></a>
				<a href=""><i class="soc6"></i></a>
				<a href=""><i class="soc7"></i></a>
			</div>
		</div>
	</div>
</div> 
<!--end footer-->
<!--begin pop-up-->
<div class="black_overlay">
</div>
<div class="pop-up">
	<div class="pop-up-wrap">
		<div class="pop-up-head clearfix">
			<h2 class="left">Регистрация</h2>
			<a href="#" class="pop-up-close right"><i></i></a>
		</div>
		<div class="pop-up-body">
			<form action="">
				<div class="top-form">
					<label for=""><span>*</span>Ваше Имя:<input type="text"></label>
					<label for=""><span>*</span>Электронная почта:<input type="email"></label>
					<label for=""><span>*</span>Придумайте себе пароль:<input type="password"></label>
				</div>
				<div class="bot-form">
					<p class="agreement">
					Регистрируясь, вы соглашаетесь с 
					<a href="#">пользовательским соглашением</a></p>
					<div class="check">
						<p><span>*</span>поля обязательные для заполнения</p>
						<label for=""><input type="checkbox">получать новости об акциях и спецпредложениях</label>
					</div>
					<input type="submit" class="reg-btn" value="Зарегистрироваться">
					<div class="pop-up-social">
						<h2>Связь с социальными сетями</h2>
						<p>Вы можете зарегистрироваться на «Розетке» с помощью учетных записей социальных сетей</p>
						<a href="#"><img src="img/pop-up-fb.jpg" height="22" width="92"></a>
						<a href="#"><img src="img/pop-up-vk.jpg"></a>
					</div>
				</div>
			</form>
		</div>
	</div>
</div> 
<!--end pop-up-->
<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
<script src="js/slick.min.js"></script>
<script src="js/main.js"></script>
</body>
</html>