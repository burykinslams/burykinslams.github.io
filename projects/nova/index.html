<!doctype html>
<html>
<head>
	<meta charset="utf-8">
	<title>Alexandr Burykin</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link href="css/screen.css" rel="stylesheet">
	<link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic&subset=latin,cyrillic-ext' rel='stylesheet' type='text/css'>
</head>
<body>
	<nav>
		<div class="container">
			<a href="#" class="hamburger">
				<div class="line"></div>
				<div class="line"></div>
				<div class="line"></div>
			</a>
			<ul class="menujs clearfix">
				<li class="is-active"><a href="#">главная</a></li>
				<li><a href="#">решение</a></li>
				<li><a href="#">опыт</a></li>
				<li><a href="#">качество</a></li>
				<li><a href="#">клиенты</a></li>
				<li><a href="#">сертификаты</a></li>
			</ul>
		</div>
	</nav>
	<header>
		<div class="container">
			<div class="logo">
				<a href="#"><img src="img/logo.png" alt=""></a>
			</div>
			<div class="description">
				<p>Ремонт и диагностика, запасные части<br>к дизельным двигатлям DEUT по всей России</p>
			</div>
			<div class="feedback">
				<p>8 (800) 333-44-29</p>
				<a href="#" class="feedback-btn">ЗАКАЗАТЬ ОБРАТНЫЙ ЗВОНОК</a>
			</div>
		</div>
	</header>
	<div class="content">
		<div class="container clearfix">
			<div class="content-banner">
				<div class="img-wrap">
					<img src="img/banner-img.png" alt="">
				</div>
			</div>
			<div class="content-info">
				<p><span>Продажа двигателей DEUT</span> а также <span>запчастей</span> к ним
				Гарантия качества сроком на 10 лет и техническая поддержка</p>
				<p class="delivery">с доставкой по всей <span>россии и странам снг!</span></p>
			</div>
			<form id="userform" class="main-form clearfix" action="">
				<label>узнайте наши цены прямо сейчас!<br>
				<span>введите Ваше имя и номер телефона в форме справа и мы Вам перезвоним</span>
				</label>
				<div class="actions">
					<input type="text" id="username" name="username" placeholder="ИМЯ">
					<input type="text" id="userphone" name="userphone" placeholder="НОМЕР ТЕЛЕФОНА">
					<input type="submit" value="УЗНАТЬ ЦЕНЫ">
				</div>
			</form>
		</div>
	</div>
<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
<script src="js/main.js"></script>
</body>
</html>	