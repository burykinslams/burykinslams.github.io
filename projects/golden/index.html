<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
	<link type="text/css" rel="stylesheet" href="style.css">
</head>
<body>
	<div id="wrapper">
<!--begin header-->
		<div id="header">
			<div class="container clearfix">
				<div id="logo">
					<a href="#"><img src="img/logo.png" height="28" width="79" alt=""></a>
				</div>
				<nav>
				    <a href="#" class="hamburger">
     					<div class="line"></div>
     					<div class="line"></div>
     					<div class="line"></div>
     				</a>
					<ul class="nav">
						<li><a href="#">home</a></li>
						<li><a href="#">service</a></li>
						<li><a href="#">portfolio</a></li>
						<li><a href="#">about</a></li>
						<li><a href="#">contact</a></li>
					</ul>
     			</nav>
			</div>
		</div>
<!--end header-->
<!--begin home-->
		<div id="home">
			<div class="container">	
				<h3>Welcome To Our Studio!</h3>
				<h1>Itâ€™s nice to Meet you</h1>
				<buttun class="btn">tell me more</buttun>
			</div>
		</div>
<!--end home-->
<!--begin services-->
		<div id="services" class="box">
			<div class="container">
				<h2 class="title">Services</h2>
				<p class="descr">Proin iaculis purus consequat sem cure. </p>
				<div class="services-wrap clearfix">
					<div class="services-item comerce">
						<h5 class="title">E-Commerce</h5>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit.</p>
					</div>		
					<div class="services-item responsive">
						<h5 class="title">Responsive Web</h5>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit.</p>
					</div>		
					<div class="services-item security">
						<h5 class="title">Web Security</h5>
						<p>Proin iaculis purus consequat sem cure
						digni ssim. Donec porttitora entum suscipit.</p>
					</div>				
				</div>
			</div>
		</div>
<!--end services-->
<!--begin portfolio-->
		<div id="portfolio" class="box">
			<div class="container">
				<h2 class="title">Our portfolio</h2>
				<p class="descr">Proin iaculis purus consequat 	sem cure. </p>	
				<div class="portfolio-wrap clearfix">
					<div class="p-item">
						<a href="#"><img src="img/p-item1.jpg" alt=""></a>
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Branding</span>
						</div>
					</div>
					<div class="p-item">
						<a href="#"><img src="img/p-item2.jpg" alt=""></a>
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Print Design</span>
						</div>
					</div>
					<div class="p-item">
						<a href="#"><img src="img/p-item3.jpg" alt=""></a>	
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Print Design</span>
						</div>
					</div>
					<div class="p-item">
						<a href="#"><img src="img/p-item4.jpg" alt=""></a>	
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Product Mock-Up</span>
						</div>
					</div>
					<div class="p-item">
						<a href="#"><img src="img/p-item5.jpg" alt=""></a>
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Web Design</span>
						</div>
					</div>
					<div class="p-item">
						<a href="#"><img src="img/p-item6.jpg" alt=""></a>
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Branding</span>
						</div>
					</div>
				</div>
			</div>
		</div>
<!--end portfolio-->
<!--begin about-->
		<div id="about" class="box">
			<div class="container">
				<h2 class="title">About Us</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div class="about-wrap">
					<div class="about-item clearfix">
						<div class="about-text left text-right">
							<h5 class="title">July 2010<br>
							Our Humble Beginnings</h5>
							<p class="descr">Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img left">
							<img src="img/a-item1.png" alt="">
						</div>
					</div>
					<div class="about-item clearfix">
						<div class="about-text right text-left">
							<h5 class="title">January 2011 <br>
							Facing Startup Battles
							</h5>
							<p class="descr">Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit aenean 
							rhoncus posuere odio in tincidunt.</p>
						</div>
						<div class="about-img right">
							<img src="img/a-item2.png" alt="">
						</div>
					</div>
					<div class="about-item clearfix">
						<div class="about-text left text-right">
							<h5 class="title">December 2012 <br>
							Enter The Dark Days
							</h5>
							<p class="descr">Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni.</p>
						</div>
						<div class="about-img left">
							<img src="img/a-item3.png" alt="">
						</div>
					</div>
					<div class="about-item clearfix">
						<div class="about-text right text-left">
							<h5 class="title">July 2010<br>
							Our Humble Beginnings</h5>
							<p class="descr">Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img right">
							<img src="img/a-item4.png" alt="">
						</div>
					</div>
					<a href="#" class="about-link"><br />Our<br />Story<br />Continues<br />...</a>
				</div>
			</div>
		</div>
<!--end about-->
<!--begin team-->
		<div id="team" class="box">
			<div class="container">
				<h2 class="title">our amazing team</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div class="team-wrap clearfix">
					<div class="team-item">
						<img src="img/team-img.png" height="220" width="220" >	
						<h5 class="title">Kimberly Thompson</h5>
						<p class="cat">Marketer</p>
						<div class="social">
							<a href="#" class="ic-tw">&nbsp</a>
							<a href="#" class="ic-fb">&nbsp</a>
							<a href="#" class="ic-pn">&nbsp</a>	
						</div>			
					</div>
					<div class="team-item">
						<img src="img/team-img.png" height="220" width="220" >	
						<h5 class="title">Rico Massimo</h5>
						<p class="cat">Coder</p>
						<div class="social">
							<a href="#" class="ic-tw">&nbsp</a>
							<a href="#" class="ic-fb">&nbsp</a>
							<a href="#" class="ic-gl">&nbsp</a>	
						</div>			
					</div>
					<div class="team-item">
						<img src="img/team-img.png" height="220" width="220" >	
						<h5 class="title">Uku Mason</h5>
						<p class="cat">Graphic Designer</p>
						<div class="social">
							<a href="#" class="ic-fb">&nbsp</a>
							<a href="#" class="ic-pn">&nbsp</a>	
						</div>			
					</div>
				</div>
				<p class="descr team-info">
					Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit <br> aenean rhoncus posuere odio in tincidunt proin iaculis.
				</p>
			</div>
		</div>
<!--end team-->
<!--begin brand-->
		<div id="brand">
			<div class="container">
			<a href="#"><img src="img/envato.jpg" height="30" width="161"></a>
			<a href="#"><img src="img/wordpress.png"  height="42" width="186"></a>
			<a href="#"><img src="img/tuts.jpg"  height="41" width="131"></a>
			<a href="#"><img src="img/microlancer.jpg"  height="42" width="167"></a>
			</div>
		</div>
		<div id="contact" class="box">
			<div class="container">
				<h2 class="title">contact us</h2>
				<p class="descr">Proin iaculis purus consequat sem cure. </p>
			<form action="#" id="message">
				<div class="clearfix">
					<div class="left">
						<input type="text" placeholder="your name *">
						<input type="text" placeholder="your e-mail *">
						<input type="text" placeholder="subject">					
					</div>
					<div class="right">
						<textarea cols="0" rows="0" <input type="text" placeholder="your message*"></textarea>
					</div>
				</div>
				<input type="submit" value="send message" class="btn">
			</form>
			</div>
		</div>
		<div id="copyright">
			<div class="container">
				<div class="social">
					<a href="#" class="ic-tw black">&nbsp</a>
					<a href="#" class="ic-fb black">&nbsp</a>
					<a href="#" class="ic-pn black">&nbsp</a>
					<a href="#" class="ic-gl black">&nbsp</a>	
				</div>	
				<p> &copy; Copyright 2014 FreebiesXpress.com</p>
			</div>
		</div>
<!--end brand-->
	</div>
	<script src="js/vendor/jquery-1.11.2.min.js"></script>
	<script src="js/main.js"></script>
</body>
</html>